### PR TITLE
refactors grid and adds some markup elemnts to allow for restyling of…

### PR DESCRIPTION
**Pull request by devin5**
… objective text to wrap from top

### CODE REVIEW

File: src/RadialTile.jsx [hunk 1]
```diff
@@ -123,15 +123,18 @@ const RadialTile = ({ time, index, objective, quote }) => {
...
+        <Grid justify="space-between" wrap="nowrap" container>
+          <div className={classes.objective} onClick={pushToHistory}>
+            {lineBreakOnSemiColon(objective)}
+          </div>
+          <div className={classes.logoContain} onClick={pushToHistory}></div>
         </Grid>
       </Grid>
       <div className={classes.radialContainer}>
-        <RadialVis time={time} index={index} />
+        <RadialVis pushToHistory={pushToHistory} time={time} index={index} />
```
Comment (danchapman) [inline src/RadialTile.jsx:134]: Might be time to change this method name to something other than `pushToHistory`

**Review by danchapman — APPROVED** [review]
Approved, with one optional comment.

File: src/RadialTile.jsx (hunk 1 above)
Comment (devin5) [reply src/RadialTile.jsx]: cool it was the same in 4 files.  I changed it to show what it is pushing to history.

-names pushToKrDahsboard  where applicable
-names pushToObjectiveDashbaord where applicable

Comment (devin5) [reply src/RadialTile.jsx]: is there an easy way to get to these. I cant seem to find any old ones unless I can remember the pr and look it up that way, sorry If I should of waiting to push this. have to get going on finishing other one...

**Pull request merged**